models/demos/ttnn_resnet/tt/ttnn_functional_resnet50_large_new_conv_api.py

```diff
@@ -590,14 +598,7 @@ def first_run(self, input_tensor, device, batch_size, ops_parallel_config) -> tt
         ]
         if is_wormhole_b0() and self.batch_size == 20:
             x, x_height, x_width = self.layer1_module1(
-                x,
-                device,
-                batch_size,
-                x_height,
-                x_width,
-                conv_op_cache,
-                reshard_if_not_optimal=True,
-                height_sharding=True,
+                x, device, batch_size, x_height, x_width, conv_op_cache, reshard_if_not_optimal=True
```

Comment:
why remove the HS flag? Isn't it used in the module?

Reply:
Default is height sharding. I removed it wherever it was hardcoded as True.